packages/dd-trace/src/priority_sampler.js

```diff
@@ -166,6 +226,13 @@ class PrioritySampler {
     }
   }
 
+  /**
+   *
+   * @param context {DatadogSpanContext}
+   * @param rule {SamplingRule}
+   * @returns {SamplingPriority}
+   * @private
```

Comment:
Instead of marking something as private, should we not just use a proper private JS property (`#privateProperty`), if it is not currently somewhere "misused" internally? :)

Reply:
Unfortunately we've already broken this by calling it [here](https://github.com/DataDog/dd-trace-js/blob/d88b6625a6ada85334b8585d302083084dabecf6/packages/dd-trace/test/standalone/tracesource_priority_sampler.spec.js#L51)

I can remove the `@private` declaration